src/languages/ruby.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  argumentMatcher,
+  conditionMatcher,
+  createPatternMatchers,
+  trailingMatcher,
+} from "../util/nodeMatchers";
+import { NodeMatcherAlternative, ScopeType, SelectionWithEditor } from "../typings/Types";
+import { SyntaxNode } from "web-tree-sitter";
+import { getNodeRange } from "../util/nodeSelectors";
+
+// Generated by the following command:
+// > curl https://raw.githubusercontent.com/tree-sitter/tree-sitter-ruby/master/src/node-types.json \  
+// | jq '[.[] | select(.type == "_statement" or .type == "_simple_statement") | .subtypes[].type]'
+const STATEMENT_TYPES = [
+  "_expression",
```

Comment:
Ok see https://github.com/Will-Sommers/cursorless-vscode/pull/1

Reply:
Heyo @pokey — I've got to confess that I feel like I'm in the deep end on this one, in terms of adding extra `EXPRESSION_STATEMENT_PARENT_TYPES`. 

ex.
```
if true
    a = 1
end
```

output:
```
[program](https://tree-sitter.github.io/tree-sitter/playground#) [0, 0] - [3, 0]
  [if](https://tree-sitter.github.io/tree-sitter/playground#) [0, 0] - [2, 3]
    condition: [true](https://tree-sitter.github.io/tree-sitter/playground#) [0, 3] - [0, 7]
    consequence: [then](https://tree-sitter.github.io/tree-sitter/playground#) [0, 7] - [1, 9]
      [assignment](https://tree-sitter.github.io/tree-sitter/playground#) [1, 4] - [1, 9]
        left: [identifier](https://tree-sitter.github.io/tree-sitter/playground#) [1, 4] - [1, 5]
        right: [integer](https://tree-sitter.github.io/tree-sitter/playground#) [1, 8] - [1, 9]
  ```

In this case, if I was to say `"take state air"`, we would expect that only that statement is highlighted, rather than the broader, if statement?

Reply:
Yeah that's right. If I'm reading this right, you need to add `then` as an allowable parent type to make this one work

Reply:
Cools, added almost all of the common cases here. Reading through the corpus on the ruby-tree-sitter project was a trip/learning experience...